package.json

```diff
@@ -54,6 +54,7 @@
     "prod:analyze": "npm run prod -- --env presets=analyze",
     "build": "npm run prod",
     "start": "npm run dev",
+    "test": "npx playwright test",
```

Comment:
Can you add these scripts too please

```
    "test": "npx playwright test",
    "test:ui": "npx playwright test --ui",
    "test:ui:codespaces": "npx playwright test --ui-host=0.0.0.0",
 ```

Reply:
Add add the following to the README:


## Running Playwright tests

To be able to run the [playwright](https://playwright.dev) tests, you first need to install the dependencies. You can see how this is done in [.devcontainer/post-create.sh](./.devcontainer/post-create.sh) which is automatically executed when opening this repository in a github codespace.

When the dependencies are set up, you can run the test suite in your terminal:

```bash
yarn test
```

To run tests visually in the playwright UI, you can use the following command:

```bash
yarn test:ui
```

This will open the playwright UI in a browser, where you can run single tests, and also inspect visually.

If you want to use the playwright UI from a github codespace, you can use this command:

```bash
yarn test:ui:codespaces
```

In general it is a good practice, and very helpful for reviewers and users of this project, that all use cases are covered in Playwright tests. Also, when contributing, try to make your tests as simple and clear as possible, so that they serve as examples on how to use the functionality.

Reply:
And just drag over .devcontainer from here

https://github.com/NEARBuilders/near-bos-webcomponent/tree/main/.devcontainer